config.toml

```diff
@@ -159,3 +159,7 @@ blog = ["HTML", "RSS"]
 baseName = "humans"
 isPlainText = true
 mediaType = "text/plain"
+
+[taxonomies]
+  tag = 'tags'
```

Comment:
Although `tags` is a default Hugo taxonomy, it had to be added here to continue to work now that the custom `authors` taxonomy has been added.